src/components/SocialLink.jsx

```diff
@@ -0,0 +1,26 @@
+const SocialLink = ({ width = 24, height = 24}) => {
+    return <div className={"flex gap-2 m-8"}>
+        <button className={"border-2 border-stone-800 rounded-full p-2 hover:bg-gray-700"}>
```

Comment:
here, in place of button you can use Link from `react-router-dom` as in future, you may want to add social link, for current placeholder you can add `<Link to="/"></Link>` you can wrap like that and add style accordingly. Do same for next button tags on this component. Thanks

Reply:
Above change is done by replacing the button tag with Link component from router-dom

Reply:
nice job